src/command_modules/vsts-cli-release/vsts/cli/release/arguments.py

```diff
@@ -0,0 +1,30 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from knack.arguments import enum_choice_list, ArgumentsContext
+
+_ON_OFF_SWITCH_VALUES = ['on', 'off']
+
+
+def load_release_arguments(cli_command_loader):
+    with ArgumentsContext(cli_command_loader, 'release') as ac:
+        ac.argument('open_browser', options_list='--open')
+        ac.argument('project', options_list=('--project', '-p'))
+        ac.argument('team_instance', options_list=('--instance', '-i'))
+        ac.argument('detect', **enum_choice_list(_ON_OFF_SWITCH_VALUES))
+
+    with ArgumentsContext(cli_command_loader, 'release list') as ac:
+        ac.argument('definition_id', nargs='*', type=int)
```

Comment:
Preferably add short names for definition-id, release-id, definition-name, release-name making it easier to type the commands. 

General rule is id and name of the main resource can be given by --name --id. e.g. release show command can have release-id as --id and release-name as --name
release definition show command can have definition-id as --id definition-name as --name.